.changeset/kind-crabs-report.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
There are lots of seemingly unrelated changes. This PR is simply the result of:

```
$ git cherry-pick 7f4232cbdb60a4475c565e5d262b25182e47baf4
$ git cherry-pick 5c1511fe1e1d1df94967623b29ec12ffe770aacf
```

And the fixing merge conflicts.